OP-138 - Patient Photo to File System Repository (#108) (#155)
informatici/openhospital-doc: doc_admin/AdminManual.adoc

image:image24.png[Login.png,width=375,height=139]

By default, USERSLISTLOGIN is set to no.

NOTE: An application restart is required to apply the modified setting.

==== 3.1.37 PATIENTPHOTOSTORAGE

The following table shows the default value and the allowed ones:

[cols=",,",options="header",]
|===
|Key |Default Value |Valid Values
|PATIENTPHOTOSTORAGE |DB |DB, _<path_to_folder>_
|===

Open Hospital can save the user's profile picture in the database or on the file system.

> NOTE: The maximum size for patient's profile picture is limited to 4096 bytes. If a bigger image is provided it will automatically resized.

Author: ```suggestion
NOTE: The maximum size for patient's profile picture is limited to 4096 bytes. If a bigger image is provided, it will automatically resized.
```

> If the PATIENTPHOTOSTORAGE parameter is set to a path (different from "DB"), Open Hospital searches for the profile picture in the specified path.

Author: ```suggestion
If the PATIENTPHOTOSTORAGE parameter is set to a path (different from "DB"), Open Hospital looks for or save the patient's profile picture in the specified path.
```

> If the PATIENTPHOTOSTORAGE parameter is set to a path (different from "DB"), Open Hospital looks for or save the patient's profile picture in the specified path.

Reviewer: Alternative wording:

If the PATIENTPHOTOSTORAGE parameter is set to a path (that is, not "DB"), then Open Hospital looks for or saves the patient's profile picture in the specified path.

Author: Done, thanks!

> If the PATIENTPHOTOSTORAGE parameter is set to a path (different from "DB"), Open Hospital looks for or save the patient's profile picture in the specified path.

Author: ```suggestion
If the PATIENTPHOTOSTORAGE parameter is set to a path (that is, not "DB"), then Open Hospital looks for or saves the patient's profile picture in the specified path.
```

> If the PATIENTPHOTOSTORAGE parameter is set to "DB", then Open Hospital searches for the patient profile picture in the DB.

Author: ```suggestion
If the PATIENTPHOTOSTORAGE parameter is set to "DB", then Open Hospital looks for or save the patient's profile picture in the DB.
```

> If the PATIENTPHOTOSTORAGE parameter is set to "DB", then Open Hospital looks for or save the patient's profile picture in the DB.

Reviewer: Alternative wording:

If the PATIENTPHOTOSTORAGE parameter is set to "DB", then Open Hospital looks for or saves the patient's profile picture in the Open Hospital database.

Author: Done, thanks!

> If the PATIENTPHOTOSTORAGE parameter is set to "DB", then Open Hospital looks for or save the patient's profile picture in the DB.

Author: ```suggestion
If the PATIENTPHOTOSTORAGE parameter is set to "DB", then Open Hospital looks for or saves the patient's profile picture in the Open Hospital database.
```
